src/main/groovy/com/pszymczyk/consul/ConsulStarter.groovy

```diff
@@ -29,9 +29,10 @@ class ConsulStarter {
     private AntUnzip unzip
 
     @PackageScope
-    ConsulStarter(Path dataDir, Path downloadDir, String consulVersion, File portsConfigFile, LogLevel logLevel, ConsulPorts ports) {
+    ConsulStarter(Path dataDir, Path downloadDir, Path configDir, String consulVersion, String customConfig, LogLevel logLevel, ConsulPorts ports) {
```

Comment:
Oh right, looks like the simplest solution is to read it from /agent/self

https://www.consul.io/docs/agent/http/agent.html#agent_self